src/components/ConfigEditor/ConnectionConfig.tsx

```diff
@@ -146,6 +149,31 @@ const ConnectionConfig: React.FC<ConnectionConfigProps> = ({
         isConfigured={!!secureJsonFields?.clientSecret}
         tooltip={clientSecretTooltip}
       />
+
+      <InlineField
+        label="Use On-Behalf-Of"
+        labelWidth={26}
+        tooltip={
+          <>
+            Propagate Grafana client credentials to ADX with a token exchange. When enabled the service account (Client
+            ID) impersonates the user by augmenting the access token. See the{' '}
```

Comment:
I like this, but the part about grafana version should be in the documentation and not in a tooltip.

Reply:
Can you be more specific on where things should be?

Reply:
You can add something like `Note: # On-Behalf-Of Authorization is only compatible with Grafana 8.3.4 or later.` in the On-Behalf-Of Authorization section in the documentation.

Reply:
What is "the documentation"? Can you be specific?

Reply:
In doc/on-behalf-of.md. Add `⚠️ Only compatible with Grafana 8.3.4 or later` above `⚠️ Grafana alert-rules do not work with on-behalf-of authorization.`. It will look a bit messy with two warnings next to each other, but we can improve documentation later.